modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -185,7 +186,7 @@ export async function getPickLists() {
 	);
 }
 
-export async function getPickListItems(pickListId: number) {
+export async function getPickListItems(pickListId: number | undefined) {
```

Comment:
This cannot be undefined, or we will get an API error. Please, do the type narrowing on the function call instead of allowing types that can cause a runtime error.

This change will cause typescript compile errors on EditObjectStateField.tsx and ModalAddFilter.tsx. You can fix them by checking (with a if statement) if there is an objectField.listTypeDefinitionId.